sale_order_type/data/default_type.xml

```diff
@@ -1,7 +1,7 @@
 <odoo noupdate="1">
     <record id="normal_sale_type" model="sale.order.type">
         <field name="name">Normal Order</field>
-        <field name="company_id" eval="False" />
+        <field name="company_id" eval="ref('base.main_company')" />
```

Comment:
Why this change?

Reply:
made it simpler to test with a db with multiple companies, but I don't mind reverting this